ytypes/int_type.go

```diff
@@ -127,7 +127,7 @@ func validateIntSlice(schema *yang.Entry, value interface{}) error {
 	return nil
 }
 
-// validateIntSchema validates the given integer type schema. This is a sanity
+// validateIntSchema validates the given integer type schema. This is a quick
```

Comment:
Same as above - throughout, it looks like this is a standard comment.

Reply:
I changed all places.